src/bootstrap/bootstrap.py

```diff
@@ -460,8 +460,14 @@ def download_stage0(self):
 
     def downloading_llvm(self):
         opt = self.get_toml('download-ci-llvm', 'llvm')
+        supported_platforms = [
+            "x86_64-unknown-linux-gnu",
+            # FIXME: maybe windows-gnu should be supported too?
+            "x86_64-pc-windows-msvc",
```

Comment:
I'd be fairly comfortable putting all T1 targets on this list at this point. x86_64 working seems good enough indication that 32-bit equivalents and the aarch64 target will work too, since the complexity is in primarily in the "os" part rather than the "arch" part of the triple.

Reply:
Ok, I added them to the list. Is there a way to see a list of all the artifacts that are uploaded? e.g. https://ci-artifacts.rust-lang.org/rustc-builds/695f18ebab0ca895841514da841eb3bd6f6bbd88